drivers/util.py

```diff
@@ -668,19 +668,22 @@ def get_all_slaves(session):
     master_ref = get_this_host_ref(session)
     return filter(lambda x: x != master_ref, host_refs)
 
-def get_nfs_timeout(session, sr_uuid):
+def get_other_config(session, sr_uuid, key):
```

Comment:
Given this function is not generic, isn't there any way to cache other config to avoid that 2 calls for
different keys will issue 2 calls to xapi?